packages/api-clients/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "lint": "eslint . --ext .ts,.tsx",
+    "lint:openapi": "bash -c 'shopt -s extglob && spectral lint ./packages/api-clients/open-api/!\\(selfcareV2ClientApi.yml\\|authorizationManagementApi.yaml\\|notifierApi.yml\\) --ruleset ../../.spectral.yaml'",
```

Comment:
Just realized that we use the spectral file only to lint stuff under api-clients, I would either move it inside `api-client`, or just run the spectral command from the root (both here and in the CI) linting all API specs in the project (at the moment, the result is the same, since we have them all under API clients)

Reply:
Also, maybe there is a way to put the ignored files in the spectral config? So that they are taken from there both from the CI and this command